android/app/src/main/java/com/integreat/MainActivity.java

```diff
@@ -12,4 +24,15 @@
     protected String getMainComponentName() {
         return "Integreat";
     }
+
+    @Override
+    public void onConfigurationChanged(Configuration newConfig) {
+        super.onConfigurationChanged(newConfig);
+
+        Locale locale = newConfig.locale;
+        if (!currentLocale.getISO3Language().equals(locale.getISO3Language())) {
+            currentLocale = locale;
+            getReactInstanceManager().recreateReactContextInBackground();
```

Comment:
So what does that do again? "Reload" the entire app and forget about the previous state?

Reply:
Yes, else the app stays in the old language on Android. Now it behaves the same on Android and iOS